packages/edit-site/src/components/error-boundary/index.js

```diff
@@ -20,6 +21,10 @@ export default class ErrorBoundary extends Component {
 		};
 	}
 
+	componentDidCatch( error ) {
```

Comment:
Did it work before? 😅 

Should it call `this.setState( { error } );` as all other error boundaries?

Actually, it looks like this is the now only component that doing it correctly by updating the state with `getDerivedStateFromError` and logging the error with `componentDidCatch` 🙃

https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries

> A class component becomes an error boundary if it defines either (or both) of the lifecycle methods [static getDerivedStateFromError()](https://reactjs.org/docs/react-component.html#static-getderivedstatefromerror) or [componentDidCatch()](https://reactjs.org/docs/react-component.html#componentdidcatch). Use static getDerivedStateFromError() to render a fallback UI after an error has been thrown. Use componentDidCatch() to log error information.

Reply:
> Did it work before? 

I actually don't know, but I can test it out!

Reply:
> Did it work before?

Yep, just checked and it did work fine. 

> Actually, it looks like this is the now only component that doing it correctly by updating the state with getDerivedStateFromError and logging the error with componentDidCatch upside_down_face

Ah, nice. I think this could be a neat little improvement to work as a follow-up :)

Reply:
👍🏻